### PR TITLE
Dark mode

### PR DESCRIPTION
**Pull request by MohammadAlhallaq**
I'm creating this draft pull request in order to make it accessible to other contributors. I would be happy to hear any suggestions regarding colors or anything else.

https://github.com/brendt/rfc-vote/assets/19909432/69abbf5e-7008-4be6-8aff-82d74131459f

### CODE REVIEW

**Comment by brendt** [comment]
As far as I can tell, it looks good. Anyone else wants to pitch in?

**Comment by SerhiiCho** [comment]
That switch button is hot 🤩 Love it... Can you change the colors of the border around vote bars as well? And the upvote button as well should be a different version when dark theme is enabled. Great change to the platform

**Comment by MohammadAlhallaq** [comment]
> That switch button is hot 🤩 Love it... Can you change the colors of the border around vote bars as well? And the upvote button as well should be a different version when dark theme is enabled. Great change to the platform

Thanks buddy, I've got a few ideas I'm going to work on today, as things started to get more clear after reviewing a few popular dark themes.

**Comment by MohammadAlhallaq** [comment]
the latest updates, Don't you think that this text color easier on the eyes


Screenshot 2023-08-26 at 1 23 26 AM
Screenshot 2023-08-26 at 1 23 40 AM
Screenshot 2023-08-26 at 1 23 49 AM
?


Screenshot 2023-08-26 at 1 23 13 AM

**Comment by SerhiiCho** [comment]
Wow! I love it. The font color is good as well

**Comment by brendt** [comment]
Very nice — within the category "dark theme" of course. Overall, this sucks. But in a good way. 👍

**Comment by MohammadAlhallaq** [comment]
I have put the final touches on this PR. Please let me know if there is anything you think could be changed or improved. I would really appreciate it.

Screenshot 2023-08-27 at 10 29 58 PM
Screenshot 2023-08-27 at 10 31 14 PM
Screenshot 2023-08-27 at 10 30 36 PM
Screenshot 2023-08-27 at 10 30 24 PM

Screenshot 2023-08-26 at 1 23 49 AM

**Comment by Erol314** [comment]
so, 
the downvote button seems odd to me, the color is too bright compared to base dark red on backround
so i changed       --color-disagree-arrow: #dd717f;

was : 
image


modified
image

**Comment by SerhiiCho** [comment]
Great! What about form inputs? They should also match dark theme

**Comment by MohammadAlhallaq** [comment]
> so, the downvote button seems odd to me, the color is too bright compared to base dark red on background so i changed a bit
> 
> was : image
> 
> modified image
> 
> not saying this is perfect choice=) but the one before was too bright compared to background red
> 
> And I think you just forgot about 'Check out repository ' button:
> 
> image
> 
> ```diff
> diff --git a/resources/css/_variables.css b/resources/css/_variables.css
> index 310fd61..abcdd25 100644
> --- a/resources/css/_variables.css
> +++ b/resources/css/_variables.css
> @@ -53,7 +53,7 @@ .dark {
>      --color-agree-arrow-background: #29392C;
>  
>      /*no arrow*/
> -    --color-disagree-arrow: #EB9CA6;
> +    --color-disagree-arrow: #dd717f;
>      --color-disagree-arrow-border: #EB9CA6;
>      --color-disagree-arrow-hover: #694048;
>      --color-disagree-arrow-background: #311820
> diff --git a/resources/views/components/footer.blade.php b/resources/views/components/footer.blade.php
> index f0a4aee..837d6bd 100644
> --- a/resources/views/components/footer.blade.php
> +++ b/resources/views/components/footer.blade.php
> @@ -8,7 +8,7 @@
>          <div class="w-full md:w-auto">
>              <a
>                  href="https://github.com/brendt/rfc-vote"
> -                class="bg-white px-3 py-2 text-main font-bold rounded hover:bg-purple-100 hover:text-main-dark"
> +                class="dark:bg-main-light px-3 py-2 text-main font-bold rounded hover:bg-purple-100 hover:text-main-dark dark:text-font"
>              >
>                  {{ __('Check out the repository') }}
>              </a>
> ```
> 
> In rest I still think its awesome=)

You're right. I think one of the trickiest parts was trying to pick the right degree for each color, which I'm not that good at.
thanks for the help!

File: resources/views/components/form/input.blade.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
         <input
             {{ $attributes->merge([
-                'class' => 'border text-gray-900 border-gray-300 rounded-md p-2 w-full ring-1 focus:ring-2 ring-transparent focus:border-purple-300 focus:ring-purple-500/50 focus:outline-none',
+                'class' => 'border text-font bg-input bg-black border-gray-300 dark:border-gray-700 rounded-md p-2 w-full ring-1 focus:ring-2 ring-transparent focus:border-purple-300 focus:ring-purple-500/50 focus:outline-none',
```
[the remark this answers is not in the document]
Comment (SerhiiCho) [reply resources/views/components/form/input.blade.php]: It seems like `bg-black` is redundant

[the remark this answers is not in the document]
Comment (MohammadAlhallaq) [reply resources/views/components/form/input.blade.php]: Yeah bg-black is totally not needed here

**Comment by brendt** [comment]
Looks good! If you fix the latest merge conflict, I'm happy to merge this PR :)

**Comment by brendt** [comment]
Looks great! Deploying right now.

Two small nitpicks I ran into: messages is a new screen, we probably need to make some changes to it?

- [ ] Cards are white
- [ ] "inbox" and "archive" title are barely readable

Screenshot 2023-08-29 at 20 06 32

**Comment by brendt** [comment]
Awesome, thanks!

**Pull request merged**